app/models/contact_email.rb

```diff
@@ -0,0 +1,4 @@
+class ContactEmail < ApplicationRecord
+  validates :email_address, presence: true, uniqueness: true, email_address: true
```

Comment:
Email shouldn't be` uniqueness: true` for this model, because contacts can share the same emails